addon/templates/components/bs-collapse.hbs

```diff
@@ -3,9 +3,9 @@
   ...attributes
   {{ref this "_element"}}
   {{style
-    width=(if (bs-eq this.collapseDimension "width") (concat this.collapseSize "px"))
-    height=(if (bs-eq this.collapseDimension "height") (concat this.collapseSize "px"))
+    width=(if (bs-eq this.collapseDimension "height") (if this.width (concat this.collapseSize "px") ""))
```

Comment:
This line *seems* wrong. Shouldn't this be like this?
```suggestion
    width=(if (bs-eq this.collapseDimension "width") (if this.collapseSize (concat this.collapseSize "px") ""))
```

Reply:
I also think we shouldn't pass an empty string.. ember-style-modifier should remove the CSS style if `undefined` is passed. So it would be:

```suggestion
    width=(if (bs-eq this.collapseDimension "width") (if this.collapseSize (concat this.collapseSize "px")))
```

`{{and}}` helper would improve readability a lot:

```hbs
    width=(if (and this.collapseSize (bs-eq this.collapseDimension "width")) (concat this.collapseSize "px"))
```

Sad that the [logical operators RFC](https://github.com/emberjs/rfcs/pull/562) seems to be stale.

Reply:
i'll fix those mistakes. but I can't rely on the `and` helper can I. That's not build in or is it?

Reply:
> but I can't rely on the `and` helper can I. That's not build in or is it?

That's right. It was more like a side note as I realized how much it would improve readability in this case. It's not available yet in ember. The RFC is not even merged yet. We don't want to introduce another dependency (ember-truth-helpers) for it.